src/Sarif/Writers/MimeType.cs

```diff
@@ -73,6 +73,8 @@ public static string DetermineFromFileExtension(Uri fileUri)
         public static readonly string Directory = "application/x-directory";
         /// <summary>The MIME type used for CSharp files.</summary>
         public static readonly string CSharp = "text/x-csharp";
+        /// <summary>The MIME type for SARIF files.</summary>
+        public static readonly string Sarif = "text/x-sarif";
```

Comment:
"application" is definitely right. I'd like us to officially register SARIF with IANA, probably as "application/sarif+json" (similar to the dozens of existing MIME types "application/foo+xml"). I'll change it to "application/json" for now.

---
In reply to: [76704516](https://github.com/Microsoft/sarif-sdk/pull/501#discussion_r76704516) [](ancestors = 76704516)